java/vector/src/main/java/org/apache/arrow/vector/complex/BaseLargeRepeatedValueViewVector.java

```diff
@@ -311,17 +311,14 @@ public void setValueCount(int valueCount) {
 
   protected int getLengthOfChildVector() {
     int maxOffsetSizeSum = offsetBuffer.getInt(0) + sizeBuffer.getInt(0);
-    int minOffset = offsetBuffer.getInt(0);
     for (int i = 0; i < valueCount; i++) {
       int currentOffset = offsetBuffer.getInt((long) i * OFFSET_WIDTH);
       int currentSize = sizeBuffer.getInt((long) i * SIZE_WIDTH);
       int currentSum = currentOffset + currentSize;
-
       maxOffsetSizeSum = Math.max(maxOffsetSizeSum, currentSum);
-      minOffset = Math.min(minOffset, currentOffset);
     }
 
-    return maxOffsetSizeSum - minOffset;
+    return maxOffsetSizeSum;
```

Comment:
@lidavidm what I have done in the first implementation is incorrect, I think the test case I had didn't tackle this case, but the C data interface tests caught it. Sorry about that.

Reply:
This need a little bit of work... I am improving it.

Reply:
I updated.